internal/services/users/user_resource.go

```diff
@@ -426,6 +445,19 @@ func userResourceCreate(ctx context.Context, d *schema.ResourceData, meta interf
 		},
 	}
 
+	if v, ok := d.GetOk("employee_type"); ok {
+		properties.EmployeeType = utils.String(v.(string))
+	}
+
+	costCenter, costCenterExists := d.GetOk("cost_center")
+	division, divisionExists := d.GetOk("division")
+	if costCenterExists || divisionExists {
+		properties.EmployeeOrgData = &msgraph.EmployeeOrgData{
+			CostCenter: utils.String(costCenter.(string)),
+			Division:   utils.String(division.(string)),
+		}
+	}
```

Comment:
As these are not Optional+Computed fields, we should set them every time even if they are blank - `d.Get()` will return the default value if a property is not configured.

Reply:
done